scripts/BONSpecialGenerator.swift

```diff
@@ -51,6 +51,13 @@ let charactersRequiringFormatStrings: Set<unichar> = [
     0x0020,
 ]
 
+// These special characters are to be excluded from the human readable string dictionary
+// They are handled outside the generated dictionary
+let charactersToExcludeFromHumanReadableStringDictionary: Set<unichar> = [
+    0x0020,
+    unichar(NSAttachmentCharacter),
```

Comment:
#nth Technically, you could have an `NSAttachmentCharacter` without an image. I'm not saying it's likely, but there should at least be defined behavior and a unit test.